### PR TITLE
docs(troubleshooting.md): Update troubleshooting.md

### PR DESCRIPTION
**Pull request by thealphadollar**
The proposal adds a drop-down list in a similar fashion as Dependencies list since it feels a little weird to have list for a continuing detail as I assume that all the three options belong to the same level of information.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -79,10 +79,12 @@ xorg-x11-fonts-misc
 ```
 </details>
 
-- Check out discussions:
-  - [#290](https://github.com/GoogleChrome/puppeteer/issues/290) - Debian troubleshooting
-  - [#391](https://github.com/GoogleChrome/puppeteer/issues/391) - CentOS troubleshooting
-  - [#379](https://github.com/GoogleChrome/puppeteer/issues/379) - Alpine troubleshooting
+<details>
+  <summary>Check out discussions</summary>
```
Comment (vsemozhetbyt) [inline docs/troubleshooting.md:83]: It seems without this empty line the links are not rendered properly.
```suggestion
  <summary>Check out discussions</summary>
  
```

**Comment by thealphadollar** [comment]
@vsemozhetbyt Thank you for the review; the changes requested by you have been implemented.

**Comment by thealphadollar** [comment]
@vsemozhetbyt Please add your consent :)

**Comment by vsemozhetbyt** [comment]
I think the suggestion can be deemed as consent per se)

**Comment by thealphadollar** [comment]
@vsemozhetbyt I meant please do the process asked by google bot so this PR can be unblocked.

**Comment by vsemozhetbyt** [comment]
Sorry, I am not sure what are the formal steps for this. If the admins define them, I will conform.

**Review by aslushnikov — APPROVED** [review]
Awesome, thank you!

**Pull request merged**